build.gradle

```diff
@@ -108,18 +134,39 @@ task executeCmake(type: Exec) {
 
     workingDir "${buildDir}/cmake"
     def prebuiltJar = null
-    if (project.hasProperty('stagingUrl')) {
+    def projectName = isFips ? project.name + "-FIPS" : project.name
+    def prebuiltJarFileName = "${projectName}-${version}-${osdetector.classifier}.jar"
```

Comment:
note for other reviewers -- from osdetector [plugin README](https://github.com/google/osdetector-gradle-plugin#usage):

> `osdetector.classifier`: classifier, which is `osdetector.os + '-' + osdetector.arch`, e.g., `linux-x86_64